<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0">
   <item>
    <layout class="QGridLayout" name="mainLayout" rowstretch="0,0,0">
     <item row="0" column="0">
      <widget class="QLabel" name="databasePathLabel">
       <property name="text">
        <string>Database path</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="databasePathLayout">
       <item>
        <widget class="QLineEdit" name="databasePath">
         <property name="placeholderText">
          <string>Using system database.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="chooseDatabasePathButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="folder">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="databasesLabel">
       <property name="text">
        <string>Databases</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="databases">
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="updateServerLabel">
       <property name="text">
        <string>Update server</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QGridLayout" name="updateServerLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="officialMirrorButton">
         <property name="text">
          <string>Official mirror</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">updateServerType</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="mirrorCombo"/>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="customServerButton">
         <property name="text">
          <string>Custom server</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">updateServerType</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="customServer"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Scan options</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="heuristics">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>854</width>
        <height>424</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Heuristics</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="sslMismatch">
         <property name="text">
          <string>SSL mismatch</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="partitionTableSize">
         <property name="text">
          <string>Alert when a partition table appears to have an incorrect size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="brokenFiles">
         <property name="text">
          <string>Identify broken executable files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="socialSecurityNumbers">
         <property name="text">
          <string>Alert when file content appears to contain social security number(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="creditCardNumbers">
         <property name="text">
          <string>Alert when file content appears to contain credit card number(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="personalData">
         <property name="text">
          <string>Alert when potentially sensitive information (e.g. personal details)  is identified.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="docPassword">
         <property name="text">
          <string>Alert when an office document cannot be scanned because it is password protected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="archivePassword">
         <property name="text">
          <string>Alert when an archive (.zip file, etc.) cannot be scanned because it is password protected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="exceedsMax">
         <property name="text">
          <string>Alert when scanned object is too large</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="macros">
         <property name="text">
          <string>Alert if Office (and other) files contain macros</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cloakedUrls">
         <property name="text">
          <string>Identify obfuscated URLs in emails</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="deepScan">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>638</width>
        <height>83</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Deep scans</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>164</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="updateServerType"/>
 </buttongroups>
</ui>
